tests/protractor.conf.js

```diff
@@ -0,0 +1,43 @@
+var packagejson = require('../package.json');
+var config = {
+    // Capabilities to be passed to the webdriver instance.
+    capabilities: {
+        'browserName': 'phantomjs',
+        'phantomjs.binary.path': require('phantomjs').path
+    },
+
+    onPrepare: function () {
+        browser.ignoreSynchronization = true;
+    },
+
+    // Options to be passed to Jasmine-node.
+    jasmineNodeOpts: {
+        showColors: true,
+        defaultTimeoutInterval: 30000
+    }
+};
+
+if (process.env.SAUCE_USERNAME && process.env.SAUCE_ACCESS_KEY) {
+    config.capabilities = null;
+    config.sauceUser = process.env.SAUCE_USERNAME;
+    config.sauceKey = process.env.SAUCE_ACCESS_KEY;
+    config.multiCapabilities = [{
+        name: 'Protractor Firefox for ' + packagejson.name + ' #' + process.env.TRAVIS_BUILD_NUMBER,
```

Comment:
I'm also not sure if we should use packagejson.name, can't we get the name somewhere else automatically? Just saying that it gets forgotten to update